package.json

```diff
@@ -18,6 +19,7 @@
     "axios": "^0.19.0",
     "core-js": "^2.6.5",
     "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
```

Comment:
`dotenv` should be a `devDependency`

Reply:
Either move to `devDependencies` or remove the `try catch` block in 'index.js`
We don't need them both.